### ISSUE TITLE
BUG: F2 cannot rename the EXcalidraw file name

### ISSUE THREAD

**Issue by jiajiala** [zsviczian/obsidian-excalidraw-plugin#1900]
F2 cannot rename the EXcalidraw file name

**Comment by zsviczian** [comment]
that seems to be a change in Obsidian. I will check with the Obsidian devs on this.

**Comment by zsviczian** [comment]
Notes on solution:

```js
app.commands.commands["workspace:edit-file-title"]
```

```js
i.app.commands.addCommand({
                    id: "workspace:edit-file-title",
                    name: Lf.commands.editFileTitle(),
                    icon: "lucide-edit-3",
                    checkCallback: function(e) {
                        var t = i.activeLeaf;
                        if (t && t.view instanceof EF)
                            return e || t.setEphemeralState({
                                rename: "all"
                            }),
                            !0
                    },
                    hotkeys: [_D([], "F2")]
                }),
```

**Comment by zsviczian** [comment]
fixed in 2.2.11
